scope/src/doctor/check.rs

```diff
@@ -179,10 +185,10 @@ impl DoctorActionRun for DefaultDoctorActionRun {
             ));
         }
 
-        let (fix_result, fix_output) = self.run_fixes().await?;
+        let (fix_result, fix_output) = self.run_fixes(prompt).await?;
 
         match fix_result {
-            i32::MIN..=-1 => {
+            i32::MIN..=-3 | NO_COMMANDS_EXIT_CODE => {
```

Comment:
I feel like the `i32::MIN..=-3` range should not return an `Ok(NoFixProvided)` as exit codes in that range are unexpected.